indico/htdocs/js/indico/jquery/declarative.js

```diff
@@ -139,6 +142,7 @@
                         url: url,
                         method: method,
                         data: params,
+                        content: $this.parent().find(content).html(),
```

Comment:
OK I see how you are using this. While it's nice for that specific usecase (not having to pass an ID to the macro) I don't think it's a good idea as it reduces reusability of this feature..